vulyk/blueprints/gamification/models/rules.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+from flask_mongoengine import Document
+from mongoengine import StringField, IntField, BooleanField
+
+from ..core.rules import Rule, ProjectRule
+
+
+class RuleModel(Document):
+    """
+    Database-specific rule representation
+    """
+    id = StringField(required=True, unique=True)
```

Comment:
Let me take a quick look at the list of rules you gave me.

wait... I don't have one!